src/app/components/edit-group-dialog/edit-group-dialog.component.ts

```diff
@@ -57,8 +78,8 @@ export class EditGroupDialogComponent implements OnInit{
 
   addMember(): void {
     this.membersFormArray.push(this.fb.group({
```

Comment:
refactor this duplicated code into a method reusable in add and load

Reply:
Too different to refactor